src/services/lnd/index.ts

```diff
@@ -87,6 +87,27 @@ export const LndService = (
     }
   }
 
+  const getChannelBalances = async (pubkey?: Pubkey) => {
+    try {
+      const lnd = pubkey ? getLndFromPubkey({ pubkey }) : defaultLnd
+      if (lnd instanceof Error) return lnd
+      const { channel_balance, inbound } = await getChannelBalance({ lnd })
+      let outbound = 0
+      const inboundBal = inbound ? inbound : 0
```

Comment:
this would do the same?

```suggestion
      const inboundBal = inbound ?? 0
```